cpp/style.rst

```diff
@@ -84,24 +84,25 @@ These tasks are made much simpler if the code is easily readable and well-docume
 
 .. _style-guide-cpp-2-2:
 
-2-2. We are writing C++17
+2-2. We are writing C++20
 -------------------------
 
-We follow the official: International Standard ISO/IEC 14882:2017(E) – Programming Language C++, without any compiler specific extensions.
+We follow the official: International Standard ISO/IEC 14882:2024(E) – Programming Language C++, without any compiler specific extensions.
```

Comment:
Isn't this 2020?

Reply:
2020 is withdrawn, 2024 is the latest ISO document.